decidim-core/lib/decidim/null_query.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+#
+# This file has been copied from https://github.com/andypike/rectify/blob/master/lib/rectify/null_query.rb
+# We have done this so we can decouple Decidim from any Virtus dependency, which is a dead project
+# Please follow Decidim discussion to understand more https://github.com/decidim/decidim/discussions/7234
+module Decidim
+  class NullQuery < Query
```

Comment:
I was just thinking, do we need the `NullQuery` for anything?

I can see that it is used as the initial value for the `Query.merge` method but couldn't we just use an empty query there?

Or if this class is absolutely necessary, I would move it inside the `Query` class as a sub-class instead of having it in its own file.

Reply:
Seems to be resolved already.